care/emr/api/viewsets/scheduling/booking.py

```
@@ -63,19 +66,23 @@ def filter_by_user(self, queryset, name, value):
     database_model = TokenBooking
     pydantic_model = TokenBookingWriteSpec
     pydantic_read_model = TokenBookingReadSpec
     pydantic_update_model = TokenBookingWriteSpec
 
     filterset_class = TokenBookingFilters
-    filter_backends = [
-        DjangoFilterBackend,
-        rest_framework_filters.OrderingFilter,
-    ]
+    filter_backends = [DjangoFilterBackend, SingleFacilityTagFilter,rest_framework_filters.OrderingFilter]
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)